paraview/xmls/ProgramBase.xml

```diff
@@ -0,0 +1 @@
+<ServerManagerConfiguration></ServerManagerConfiguration>
```

Comment:
same here (this vtk-layer class should only be used by standalone programs).
I think you can remove that.